### ISSUE TITLE
Editor: Alt Text missing in Calypso when added from mobile

### ISSUE THREAD

**Issue by xpurichan** [Automattic/wp-calypso#26244]
#### Steps to reproduce
1. Starting with WordPress application on the iPhone (I tested with iPhone X and WP app version 10.3.2)
2. Create a blog post with an image, and add Alt Text to that image
3. Save changes by publishing or saving the post as a draft
4. Notice that the Alt Text does save here in the app
5. Next, load WordPress.com on a browser on PC
6. Using Calypso visit the post editor (Visual tab) where the image was uploaded
7. Click the image and edit it, notice that the Alt Text field is empty

Extra:
8. Now view the HTML tab in this editor and notice that the Alt Text is actually populated on the image
9. Visit WP Admin and notice that the Alt Text for the image is present in the Visual and Text tabs of the post editor

#### What I expected
I expect for the Alt Text field to be populated in Calypso's Visual post editor

#### What happened instead
The Alt Text field in Calypso's Visual post editor is blank even when it shows to be filled in when viewing this option from other places.

#### Screenshot / Video
Screenshots of Alt Text field in various places:
[iOS app](https://cloudup.com/cXUZuPJm82D)
[Calypso Visual editor](https://cloudup.com/cbtneVtm38X) - Missing!
[Calypso HTML editor](https://cloudup.com/cRUny_2dTEb)
[WP Admin Visual editor](https://cloudup.com/c45upI7NB1z)

#### Context / Source
User reported on Zen-1296121
I verified and it's my test and post screenshots which are reported above
